client/pages/sections/protocols/steps.js

```diff
@@ -242,14 +243,17 @@ class Step extends Component {
           }
           <h3>
             {`Step ${index + 1}`}
-            {pdf && values.reference && (<Fragment>: { values.reference }</Fragment>)}
+            {(pdf || readonly) && values.reference && (<Fragment>: { values.reference }</Fragment>)}
             {
               completed && !isUndefined(values.optional) &&
               <span className="light smaller">{` (${values.optional === true ? 'optional' : 'mandatory'})`}</span>
             }
+            {
+              !pdf && readonly && repeatedFrom && <div className="light smaller">{`Repeated from protocol ${repeatedFrom}`}</div>
+            }
           </h3>
           {
-            pdf && repeatedFrom && <span className="review"><p className="grey">{`Repeated from protocol ${repeatedFrom}`}</p></span>
+            pdf && !readonly && repeatedFrom && <span className="review"><p className="grey">{`Repeated from protocol ${repeatedFrom}`}</p></span>
```

Comment:
I'm not sure where the props come from, but `pdf && !readonly` implies to me that we're in an editable PDF state, which I find confusing.

Reply:
Removed, the `!readonly` wasn't necessary when it is a pdf, and as you say it is confusing.